examples/Makefile

```diff
@@ -1,17 +1,9 @@
 .PHONY: opencv object_detection delete minikube gatk hyperparameter
 
-JQ := jq
-JQFLAGS :=
-DOCKER := docker
-ECHO := echo
-TOUCH := touch
-CP := cp
-RM := rm
-STATS := .enable_stats=true
-DETECTMODEL := .pipeline.name="object-detect"|.input.cross[1].pfs.repo="object-model"|.input.cross[1].pfs.name="model"
-HYPERPSPLIT := .input.cross[0].pfs.repo="hp-split"|.input.cross[0].pfs.name="split"
-HYPERPMODEL := .input.cross[1].pfs.repo="hp-model"|.input.cross[1].pfs.name="model"
-HYPERPTEST := .input.cross[0].pfs.repo="hp-test"|.input.cross[0].pfs.name="test"
+PACHCTL := /usr/local/bin/pachctl
+TAR := /usr/bin/tar
+WGET := /usr/local/bin/wget
+RM := /bin/rm
```

Comment:
Why set these to the specific executable paths? This won't work on, e.g. my linux desktop because:

1) `wget` is in `/usr/bin/wget`
2) `pachctl` is in `GOPATH`

Reply:
The problem is, for testing, PACHCTL may be in a variety of places, like when testing building a cluster in 1.9.12 and restoring it in 1.10.0, and confirming the same cluster builds properly in 1.10.0.  

I think I should set them to just the raw exectuable names, and I'll do that in an update.

I do want to add logic in a future version (outlined in the todo at top) to automatically set them to platform-specific versions.  I also  note, in Makefile.testing, how to override them when you invoke it.  I think I should add that here, as well.

Reply:
If you're looking to ensure a specific version of pachctl, and ensure you're always calling the correct path, you could:

1) first call `etc/fetch_release_pachctl.py`, e.g. `etc/fetch_release_pachctl.py 1.9.12`
2) reference the absolute path to `pachctl` in `GOPATH`

Reply:
Oh, damn, that's awesome!  Yes, I'll add that.

Reply:
OK, the paths are now neutral executables and I've added your instructions above.